promql/promqltest/testdata/native_histograms.test

```diff
@@ -1319,3 +1319,60 @@ eval instant at 10m histogram_sub_3{idx="0"} - ignoring(idx) histogram_sub_3{idx
     {} {{schema:0 count:-30 sum:-1111.1 z_bucket:-2 z_bucket_w:0.001 buckets:[-1 0 -1 -2 -1 -1 -1] n_buckets:[0 2 -2 -2 -7 0 0 0 0 -5 -5 -2]}}
 
 clear
+
+# Test native histogram with trim operators ("</": TRIM_UPPER, ">/": TRIM_LOWER)
+load 1m
+  h_test {{schema:0 count:34 z_bucket:1 z_bucket_w:0.001 buckets:[2 4 8 16] n_buckets:[1 2]}} 
+  h_test_2 {{schema:2 count:27 z_bucket:1 z_bucket_w:0.001 buckets:[1 2 4 7 3] n_buckets:[1 5 3 1]}}
+  cbh {{schema:-53 sum:5 count:15 custom_values:[5 10 15 20] buckets:[1 6 4 3 1]}}
+  zero_bucket {{schema:0 z_bucket:5 z_bucket_w:0.01 buckets:[2 3] n_buckets:[1 2 3]}}
```

Comment:
To elaborate on my comment:

These should all have `sum:...` fields with plausible values.

`cbd` already has a `sum:...` field, but its value is not plausible. If we assume that the observations are more or less in the middle of each bucket (and around 2.5 for the first bucket and not much above 20 for the +Inf bucket), the sum should be around 170.